card/spec/models/card/create_spec.rb

```diff
@@ -13,14 +13,14 @@
 describe Card, "created by Card.new " do
   before(:each) do
     Card::Auth.as_bot do
-      @c = Card.new :name=>"New Card", :content=>"Great Content"
+      @c = Card.new name: "New Card", content: "Great Content"
     end
   end
 
   it "should not override explicit content with default content" do
     Card::Auth.as_bot do
-      Card.create! :name => "blue+*right+*default", :content => "joe", :type=>"Pointer"
-      c = Card.new :name => "Lady+blue", :content => "[[Jimmy]]"
+      Card.create! name: "blue+*right+*default", content: "joe", type: "Pointer"
+      c = Card.new name: "Lady+blue", content: "[[Jimmy]]"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.